assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -705,10 +705,15 @@ form {
   transition: box-shadow .2s ease-in-out;
 }
 
-.form__input:hover {
+.form__input:not(:disabled):hover {
   box-shadow: 0 0 0 3px #ededed;
 }
 
+.form__input:disabled {
+  background: #ddccdd;
+  color: #757575;
```

Comment:
are there guardian colours we can use here?

Also, is the colour change from Iona's design intentional?

Reply:
This colour isn't quite right. On my design the background is #DCDCDC. Please could we change this? (btw font should be #333333). I imagine you might have changed it for accessibility reasons, that was my mistake as it would have failed with my lighter font colour. I've checked this now and it is good from a G design perspective and an accessibility one.